app/models/certification.rb

```diff
@@ -24,12 +30,19 @@ def self.to_csv (attributes)
   end
 
   def unique_cert
-    @user_certs = self.user.certifications
-    @user_certs.each do |cert|
-      if cert.training == self.training
-        errors.add(:string, "Certification already exists.")
+    @user_certs = self.user.certifications rescue nil
```

Comment:
I'm not a fan of these rescue statements because if catches _all_ exception, not just nil exceptions. See [this blog post](http://www.nickdesteffen.com/blog/why-rescue-nil-is-bad). You can do an `return true unless user`. Also, when who this happen that a certification has no user?

Reply:
This shouldn't happen but I didn't want anything breaking in the unlikely event that this happens